packages/api/src/graphql/blocks.ts

```diff
@@ -169,9 +175,31 @@ export const GraphQLPageTeaser = new GraphQLObjectType<PageTeaser, Context>({
   isTypeOf: createProxyingIsTypeOf(value => value.type === TeaserType.Page)
 })
 
+export const GraphQLCustomTeaser = new GraphQLObjectType<CustomTeaser, Context>({
+  name: 'CustomTeaser',
+  fields: () => ({
+    style: {type: GraphQLNonNull(GraphQLTeaserStyle)},
+
+    image: {
+      type: GraphQLImage,
+      resolve: createProxyingResolver(({imageID}, {}, {loaders}) =>
+        imageID ? loaders.images.load(imageID) : null
+      )
+    },
+
+    preTitle: {type: GraphQLString},
+    title: {type: GraphQLString},
+    lead: {type: GraphQLString},
+    contentUrl: {type: GraphQLString},
```

Comment:
This is optional?

Reply:
No. Definition is ofc required. I understand to make it non-optional you need specify that as:
`{type: GraphQLNonNull(GraphQLString)}`
instead of
`{type:GraphQLString}`